### PR TITLE
Docs Renaming

### PR DESCRIPTION
**Pull request by awholegunch**
Renames all the docs from "Loom" to "Coopr", and changes "Continuuity" to "Cask Data". Includes updates to illustrations and adding "©" to copyright phrases.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Continuuity Loom CHANGELOG
-==========================
+Cask Coopr CHANGELOG
```
Comment (albertshau) [inline CHANGELOG.md]: just Coopr

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,8 @@ Unreleased
 - Added additional info in get clusters API ( Issues: #401 #430 )
 - UI: fix when a provider type has no admin fields ( Issues: #409 )
 
+As of version 0.9.8, the name of the product was changed to "Cask Coopr".
```
Comment (albertshau) [inline CHANGELOG.md]: just Coopr

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,17 +1,23 @@
-# Contributing to Continuuity Loom
+# Contributing to Cask Coopr
```
Comment (albertshau) [inline CONTRIBUTING.md]: just Coopr

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
 
-Are you interested in making Continuuity Loom better? Our development model is a simple pull-based model with a consensus building phase, similar to the Apache's voting process. If you want to help make Continuuity Loom better, by adding new features, fixing bugs, or even suggesting improvements to something that's already there, here's how you can contribute:
+Are you interested in making Cask Coopr better? Our development model is a simple
```
Comment (albertshau) [inline CONTRIBUTING.md]: just Coopr

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,17 +1,23 @@
...
+pull-based model with a consensus building phase, similar to the Apache's voting process.
+If you want to help make Cask Coopr better, by adding new features, fixing bugs, or
```
Comment (albertshau) [inline CONTRIBUTING.md]: just Coopr

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,17 +1,23 @@
...
  * Once addressed, the changes will be committed to the continuuity/loom repo.
 
-Bugs and suggestions should be made by filing a [GitHub Issue](https://github.com/continuuity/loom/issues) against this repository.
+Bugs and suggestions should be made by filing a [GitHub Issue](https://github.com/caskco/coopr/issues)
+against this repository.
 
 ## License
 
-By contributing to Continuuity Loom, you agree that your contributions will be licensed under the [Apache License Version 2.0 (APLv2)](LICENSE).
+By contributing to Cask Coopr, you agree that your contributions will be licensed under the
```
Comment (albertshau) [inline CONTRIBUTING.md]: just Coopr

File: README.md [hunk 7]
```diff
@@ -1,42 +1,53 @@
-## [Continuuity](http://www.continuuity.com) Loom : Modern Cluster Management
+## [Cask](http://cask.co) Coopr: Modern Cluster Management
 
-Continuuity Loom is cluster management software that manages clusters on public, private clouds. Clusters created with Continuuity Loom utilize templates of any hardware and software stack, from simple standalone LAMP-stack servers and traditional application servers like JBoss, to full Apache Hadoop clusters comprised of thousands of nodes. Clusters can be deployed across many cloud providers (Rackspace, Joyent, and OpenStack) while utilizing common SCM tools (Chef and scripts).
+Cask Coopr is cluster management software that manages clusters on public, private
```
Comment (albertshau) [inline README.md]: same comment about Cask Coopr -> Coopr across the readme

File: docs/docs/source/faq/server.rst [hunk 8]
```diff
@@ -110,14 +110,13 @@ Is there anyway to inspect the plan for cluster being provisioned?
...
 before proceeding to the next stage. This ensures the actions are executed in the right dependency order.
 
-Following is an example web service call along with the output returned from the Loom Server provisioning a web server
-on a single node.:
-::
+Following is an example web service call along with the output returned from the Coopr Server provisioning a web server
+on a single node::
 
-  $ curl -H 'X-Loom-UserID:<user id>' http://<loom-host-name>:<loom-host-port>/v1/loom/clusters/<cluster-id>/plans
+  $ curl -H 'X-Coopr-UserID:<user id>' http://<coopr-host-name>:<coopr-host-port>/v1/coopr/clusters/<cluster-id>/plans
```
Comment (albertshau) [inline docs/docs/source/faq/server.rst]: Looks like I missed this when updating docs.  the url should be 

```
http://<server>:<port>/v2/clusters/<cluster-id>/plans
```

File: docs/docs/source/guide/admin/ui.rst [hunk 9]
```diff
@@ -31,7 +31,7 @@ Each screen in the administration interface provides ways to create and edit set
 Login as an Administrator
 =========================
 
-The Loom Admin UI can be accessed at ``http://<loom-host>:<loom-ui-port>/``. Login using the admin credentials to access the administrator interface.
+The Coopr Admin UI can be accessed at ``http://<coopr-host>:<coopr-ui-port>/``. Login using the admin credentials to access the administrator interface.
```
Comment (albertshau) [inline docs/docs/source/guide/admin/ui.rst]: we can replace 'Coopr Admin UI' with just 'Admin UI'

File: docs/docs/source/guide/admin/ui.rst [hunk 10]
```diff
@@ -57,7 +57,7 @@ Clicking on the name of each element allows an administrator to enter its manage
 Managing Cluster Templates
 ============================
 
-Loom templates allow the administrator to define blueprints describing how different types of clusters should be laid out.
+Coopr templates allow the administrator to define blueprints describing how different types of clusters should be laid out.
```
Comment (albertshau) [inline docs/docs/source/guide/admin/ui.rst]: 'Coopr templates' can be replaced with just 'Templates'

**Comment by albertshau** [comment]
a few small comments, otherwise looks good :+1:

**Comment by awholegunch** [comment]
Fixed all comments (and then some!) Please check again.

**Comment by albertshau** [comment]
:+1:

**Pull request merged**